packages/tables/resources/views/index.blade.php

```diff
@@ -231,7 +232,7 @@ class="fi-ta-header-toolbar flex items-center justify-between gap-3 px-4 py-3 sm
                         {{ \Filament\Support\Facades\FilamentView::renderHook('tables::toolbar.search.before', scopes: static::class) }}
 
                         @if ($isGlobalSearchVisible)
-                            <x-filament-tables::search-field />
+                            <x-filament-tables::search-field :search-placeholder="$searchPlaceholder" />
```

Comment:
Can just call the getter in here if we're only using it in a single place anyway.